packages/office-ui-fabric-react/src/common/_themeOverrides.scss

```diff
@@ -1,72 +1,49 @@
-$ms-color-themeDarker: "[theme:themeDarker, default: #004578]";
```

Comment:
none of these $ms-color-*s can be remove, they're used all over the place. I suspect we're not getting compiler errors because it's falling back to the fabric core non-themed values.

why are they getting prefixed with ms-font instead of ms-color?

anyways, the diff here should be minimal, as with the above file

Reply:
That was my mistake! Just changed the script to prepend the string with "ms-color-" instead of "ms-font"

Reply:
And just to re-iterate the question I left in my PR description above:

There are some variables missing from the new _themeOverrides.scss file auto-generated from the defaults. These aren't in the default theme and were previously manually added to the old _themeOverrides.scss file. Should we add them to DefaultPalette?

Reply:
Maybe not... but run the automation that links to odsp-next and verify. I remember a few months ago David was talking about some automation that hooks up your local OUFR and odsp-next (maybe odsp-common too?) repo. Maybe @joschect can help out here.

Reply:
Let's chat in the AM @natalieethell. I have a contact for this tool and need to follow-up. Thanks @phkuo 🙇

Reply:
Just for some more context on this, it looks like a lot of these color variables (if not all) missing from the auto-generated _themeOverrides.scss are colors that are defined in _semanticSlots.scss, such as errorBackground, warning, warningBackground, etc. They have different variable names, though, and there is still code in the office-ui-fabric-react repo that uses the old variable names as appear in the old _themeOverrides.scss. So it looks like we need to keep these or change out all of those variables and do the odsp-next/odsp-common check.

Reply:
I placed the variables that are not auto-generated from IPalette into a file called _legacyThemePalette.scss. Once all references to these—what I'm assuming are old—variables are removed, we should be able to remove this file.